<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWin</class>
 <widget class="QWidget" name="RegisterWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>572</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 主窗口样式 */
RegisterWin {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                              stop:0 #f5f7fa, stop:1 #e4e8f0);
    font-family: &quot;微软雅黑&quot;;
	font-size: 18px; /* 基础字体放大 */
}

/* 表单容器样式 */
QVBoxLayout, QHBoxLayout, QGridLayout {
    margin: 15px;
    spacing: 15px;
}

/* 标签样式 */
QLabel {
	font-size: 20px; /* 从16px增大到20px */
    color: #4a5568;
    min-width: 80px; /* 增大最小宽度 */
    padding: 8px 0; /* 增大内边距 */
    font-weight: 500; /* 中等粗细 */
}

/* 输入框样式 */
QLineEdit, QDateEdit {
 	font-size: 18px; /* 从15px增大到18px */
    padding: 12px 15px; /* 增大内边距 */
    border: 1px solid #cbd5e0;
    border-radius: 6px; /* 增大圆角 */
    background: white;
    min-height: 30px; /* 增大最小高度 */
}

QLineEdit:focus, QDateEdit:focus {
    border: 2px solid #4299e1;
    
}

/* 单选按钮样式 */
QRadioButton {
    font-size: 22px;
    color: #4a5568;
    padding: 8px 15px;
    spacing: 8px;
}

QRadioButton::indicator {
    width: 18px;
    height: 18px;
    border-radius: 9px;
    border: 2px solid #a0aec0;
}

QRadioButton::indicator:checked {
    background-color: #4299e1;
    border: 2px solid #4299e1;
}

QRadioButton::indicator:hover {
    border: 2px solid #718096;
}

/* 按钮基础样式 */
QPushButton {
    font-size: 22px;
    padding: 14px 25px;
    border: none;
    border-radius: 6px;
    color: white;
    background-color: #4299e1;
    min-width: 100px;
}

/* 注册按钮特殊样式 */
QPushButton#registerBT {
    background-color: #48bb78;
}

QPushButton#registerBT:hover {
    background-color: #38a169;
}

QPushButton#registerBT:pressed {
    background-color: #2f855a;
}

/* 重置按钮特殊样式 */
QPushButton#resetBT {
    background-color: #e53e3e;
}

QPushButton#resetBT:hover {
    background-color: #c53030;
}

QPushButton#resetBT:pressed {
    background-color: #9b2c2c;
}

/* 头像区域样式 */
#headpicLB {
    background-color: #edf2f7;
    border: 2px dashed #cbd5e0;
    border-radius: 8px;
}

/* 头像操作按钮样式 */
QPushButton#addpicBT, QPushButton#videoswitchBT, QPushButton#cameraBT {
    background-color: #718096;
    font-size: 18px;
    padding: 8px 12px;
}

QPushButton#addpicBT:hover, QPushButton#videoswitchBT:hover, QPushButton#cameraBT:hover {
    background-color: #4a5568;
}

QPushButton#addpicBT:pressed, QPushButton#videoswitchBT:pressed, QPushButton#cameraBT:pressed {
    background-color: #2d3748;
}

/* 图片路径输入框 */
#lineEdit_picFileEdit {
    font-size: 14px;
    padding: 6px 10px;
    border: 1px solid #cbd5e0;
    border-radius: 4px;
    background: white;
}
&lt;/string&gt;
&lt;/property&gt;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mrb">
         <property name="text">
          <string>男</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wrb">
         <property name="text">
          <string>女</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>生日：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="birthdayEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addressEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>电话：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="registerBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,1,1">
     <item row="0" column="0">
      <widget class="QLabel" name="headpicLB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="picFileEdit"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addpicBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>添加头像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="videoswitchBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>打开摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cameraBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>拍照</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
